docs/030_user-guide/015_sdk.md

```diff
@@ -47,7 +47,7 @@ let result = containers(peprValidationRequest, "ephemeralContainers")
 
 ## `getOwnerRefFrom`
 
-Returns the owner reference for a Kubernetes resource. Accepts the following parameters:
+Returns the owner reference for a Kubernetes resource as an array. Accepts the following parameters:
 
 - **@param kubernetesResource: GenericKind** The Kubernetes resource to get the owner reference for
```

Comment:
add new params that are added
  
  blockOwnerDeletion?: boolean,
  controller?: boolean,